File: tasks/Gogacz/dodawanie.js
// JS

class Podawak {
  constructor(wartosc) {
    this.wartosc = wartosc
  }

  Podaj() {
    return this.wartosc;
  }
}

function Suma(a, b) {
  return a.Podaj() + b.Podaj()
}

File: tasks/Sebastian-Marciniak/dodawanie.py
tablica = [0, 1, 2, 3]
class Podawak:
	def __init__(self, value):
		self.value = value

	def podaj(self):
		return self.value

def dodawanie(a, b):
		return tablica[a.podaj()] + tablica[b.podaj()]